Europe Flags.style

```diff
@@ -0,0 +1,52 @@
+<img src="https://www.countries-ofthe-world.com/flags-normal/flag-of-Albania.png" width="500" height="300">
```

Comment:
Could we just have the URLs entered into the JavaScript object. For example:

```
const theEarthObjectJS = {
  continents: {
    africa: {
      "wikipedia_link": "https://en.wikipedia.org/wiki/Africa",
      "nations": {
        "angola": {
          "flag": "https://www.countries-ofthe-world.com/flags-normal/flag-of-Albania.png"
        }
      }
    },
```

Reply:
Did I update the error correctly? Or am I still missing something?

Reply:
@iarobinson